Project.toml

```diff
@@ -32,7 +32,7 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 ZygoteRules = "700de1a5-db45-46bc-99cf-38207098b444"
 
 [compat]
-AbstractMCMC = "2.1"
+AbstractMCMC = "2.1, 3"
```

Comment:
```suggestion
AbstractMCMC = "3"
```

This should cut out AbstractMCMC 2 because the internal API changes.

Reply:
But this is only relevant for users that use custom callbacks? I.e., for Turing it should be fine to support both versions since we do not depend on these internals, do we?

Reply:
Hm, I suppose not. IGNORE ME

![](https://media1.tenor.com/images/fdd9efddce00bd88302fdcfd3273673f/tenor.gif)

Reply:
Yeah, leaving it as is:)